modules/dev-server/main.tf

```diff
@@ -435,11 +437,28 @@ resource "aws_security_group" "load_balancer" {
   }
 }
 
+data "aws_security_group" "vpc_default" {
+  vpc_id = var.vpc_id
+
+  filter {
+    name   = "group-name"
+    values = ["default"]
+  }
+}
+
 resource "aws_security_group" "ecs_service" {
   count  = var.lb_enabled ? 1 : 0
   name   = "${var.name}-ecs-sg"
   vpc_id = var.vpc_id
 
+  ingress {
+    description     = "Access to Consul dev server from VPC default security group"
+    from_port       = 0
+    to_port         = 0
+    protocol        = "-1"
+    security_groups = [data.aws_security_group.vpc_default.id]
+  }
```

Comment:
todo: Remove these changes to the dev-server. They are taken from https://github.com/hashicorp/terraform-aws-consul-ecs/pull/76